boa/src/builtins/json/mod.rs

```diff
@@ -65,10 +67,69 @@ pub fn parse(_: &mut Value, args: &[Value], _: &mut Interpreter) -> ResultValue
 ///
 /// [spec]: https://tc39.es/ecma262/#sec-json.stringify
 /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify
-pub fn stringify(_: &mut Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
-    let obj = args.get(0).expect("cannot get argument for JSON.stringify");
-    let json = obj.to_json().to_string();
-    Ok(Value::from(json))
+pub fn stringify(_: &mut Value, args: &[Value], interpreter: &mut Interpreter) -> ResultValue {
+    let object = args.get(0).expect("cannot get argument for JSON.stringify");
```

Comment:
```suggestion
    let object = match args.get(0) {
        Some(object) => object,
        None => return Ok(Value::undefined()),
    };
```
Calling `JSON.stringify()` should not panic if no value has supplied

Reply:
:heavy_check_mark: